ovirt-host.spec

```diff
@@ -97,15 +97,18 @@ This meta package pulls in all the dependencies needed for an oVirt hosts.
 Summary:	This meta package pulls in all the dependencies needed for minimal oVirt hosts
 %ifnarch s390x
 # Not available for s390x yet
-Requires:	collectd
-Requires:	collectd-disk
-Requires:	collectd-netlink
-Requires:	collectd-write_http
-Requires:	collectd-virt
-
-%if 0%{?rhel}
-# collectd-write_syslog is available only on RHEL and similar
-Requires:	collectd-write_syslog
+%if %{rhv_build}
+Requires:	collectd >= 5.11.0-10
+Requires:	collectd-postgresql >= 5.11.0-10
+Requires:	collectd-disk >= 5.11.0-10
+Requires:	collectd-write_http >= 5.11.0-10
+Requires:	collectd-write_syslog >= 5.11.0-10
+%else
+Requires:	collectd >= 5.12.0-7
```

Comment:
This is going to fail on CentOS Stream 9 as https://cbs.centos.org/koji/buildinfo?buildID=37312 is fully tagged for el8s but https://cbs.centos.org/koji/buildinfo?buildID=37311 is tagged only candidate.
@mrunge anything preventing el9s build to go to testing / release ?

Reply:
oh wow. I would recommend to remove all the versioned requirements, this often causes more issues than it really solves. Note that you can not have two versions of the same package in the same repository.

A good practice is to require capabilities. E.g if you require collectd-write_syslog, then it will be only satisfied by a build providing that capability.